pyquil/paulis.py

```diff
@@ -710,15 +710,21 @@ def commuting_sets(pauli_terms):
     return groups
 
 
-def is_identity(term):
+def is_identity(pauli_object):
```

Comment:
in general, you should leave names in function signatures alone for backwards compatibility. Just in case there's code in the wild that uses
```python
is_identity(term=my_variable)
```

Reply:
also cc #542

Reply:
The argument name was changed to make `is_identity(..)` read similar to `is_zero(..)`, but I see the point -- changed the argument name back to `term`.

Reply:
Thanks!